pages/workflow%3A meetnetimplementatie.md

```diff
@@ -0,0 +1,8 @@
+tags:: guidelines, implementation, copy guidelines
+
+- Voor een nieuw meetnetontwerp kan je dit repo op de één of andere manier kopiëren (bv via een "fork").
+  Nadien is het mogelijk om concrete aanvullingen van de implementatie van een meetnet daarin op te slaan, bv. via de "journal".
+	- instructies over "forking": https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo
+	- Het is aangeraden om [de publicatie-functie nadien af te zetten]([[deactivate website publishing]]).
+	- Alternatief kan je een versie van deze guidelines downloaden en lokaal bewerken.
+- Informatie die van algemeen belang is voor het meetnetontwerp mag je graag aanvullen op https://github.com/inbo/guidelines-monitoring-networks, via een issue of pull request. Alvast dank!
```

Comment:
```suggestion
- Informatie die van algemeen belang is voor het meetnetontwerp, mag je aanvullen op https://github.com/inbo/guidelines-monitoring-networks, via een issue of pull request. Alvast dank!
```
ofwel: 
```suggestion
- Graag informatie van algemeen belang voor het meetnetontwerp aanvullen op https://github.com/inbo/guidelines-monitoring-networks, via een issue of pull request. Alvast dank!
```
'informatie die van algemeen belang is' zou je ook kunnen vervangen door 'belangrijke informatie', 'van algemeen belang' wordt naar mijn gevoel bijna enkel gebruikt in de context dat iets belangrijk is voor de hele bevolking.